crates/ruff_linter/src/rules/flake8_bugbear/snapshots/ruff_linter__rules__flake8_bugbear__tests__B006_B006_5.py.snap

```diff
@@ -151,7 +151,8 @@ B006_5.py:35:59: B006 [*] Do not use mutable data structures for argument defaul
    35 |+def import_docstring_module_wrong(value: dict[str, str] = None):
 36 36 |     """Docstring"""
 37 37 |     import os
-   38 |+    if value is None:
-   39 |+        value = {}
+   38 |+
```

Comment:
This line was added when implementing the `locator.full_line_end(statement.end()) == locator.text_len()` case. 

I can see something similar happens when doing it with docstrings on `0.0.291`.

This will:
``` python
def t(foo=[]):
    """Docstring"""

a = 2
```

will be fixed to
``` python
def t(foo=None):
    """Docstring"""
    if foo is None:
        foo = []

a = 2
```

Whereas 

``` python
def t(foo=[]):
    """Docstring"""
```
will be fixed to
``` python
def t(foo=None):
    """Docstring"""

    if foo is None:
        foo = []
```